packages/rest/samples/docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+version: '3.8'
+
+volumes:
+  postgres:
+
+services:
+  credo:
+    image: ghcr.io/openwallet-foundation/credo-rest:0.10.0
+    depends_on:
+      - postgres
+    ports:
+      # Admin API
+      - 3000:3000
+
+      # DIDComm
+      - 5000:5000
```

Comment:
FYI MacOS uses port 5000 as the default for the AirPlay receiver which has caused me weirdness in the past

Reply:
thanks, keep forgetting about it